src/stories/components/BasicModal/ContainerModal.tsx

```diff
@@ -0,0 +1,170 @@
+import styled from '@emotion/styled';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import React from 'react';
+import { Close } from '../svg/close';
+import CategoryItem from './CategoryItem/CategoryItem';
+import CheckBoxDescription from './ChekBoxDescription/ChekBoxDescription';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+
+interface Props {
+  headCountCategories: string[];
+  noHeadCountCategories: string[];
+}
+
+const ContainerModal: React.FC<Props> = ({ headCountCategories, noHeadCountCategories }) => {
+  const { isLight } = useThemeContext();
+
+  return (
+    <Container isLight={isLight}>
+      <Header isLight={isLight}>
+        <ContainerTitle>
+          <Title isLight={isLight}>Canonical Expense Categories</Title>
+          <Close />
+        </ContainerTitle>
+        <ContainerDescription>
+          <Description isLight={isLight}>
+            Canonical Expense Categories in MakerDAO are standardized classifications of expenses tailored for managing
+            MakerDAO's operational costs.
+          </Description>
+          <CheckBoxDescription />
+        </ContainerDescription>
+      </Header>
+      <InsideModal>
+        <HeadCount isLight={isLight}>Headcount Expense Categories</HeadCount>
+        <Line isLight={isLight} />
+        <HeadCountList>
+          {headCountCategories?.map((item) => (
+            <CategoryItem category={item} />
+          ))}
+        </HeadCountList>
+        <NoHeadCount isLight={isLight}>Non-Headcount Expense Categories </NoHeadCount>
+        <Line isLight={isLight} />
+        <ContainerTowColumns>
+          <ContainerPar>
+            {noHeadCountCategories
+              ?.filter((_, index) => index % 2 === 0)
+              .map((item) => (
+                <CategoryItem category={item} />
+              ))}
+          </ContainerPar>
+          <ContainerOdd>
+            {noHeadCountCategories
+              ?.filter((_, index) => index % 2 !== 0)
+              .map((item) => (
+                <CategoryItem category={item} />
+              ))}
```

Comment:
I strongly recommend implementing this using CSS columns in a way each category represents a column, it will be easier to implement responsive, to integrate with the API and more efficient than "manually" create the columns. You can take a look how bootstrap columns o MUI Grid works in columns

Reply:
You are right , done